### ISSUE TITLE
Bounding Box for path does not seem right

### ISSUE THREAD

**Issue by davibarreira** [JuliaGraphics/Luxor.jl#213]
Hey @cormullion , it seems that the BoundingBox function is not properly defining the bound box for curves / paths.
```julia
d = Drawing(100,100,:svg)
origin()
newpath()
curve(O, O+Point(10,40), O+Point(40,-30))
setcolor("blue")
box(BoundingBox(storepath()),:fill)

newpath()
curve(O, O+Point(10,40), O+Point(40,-30))
setcolor("black")
strokepath()
finish()
d
```
Here is what I get:
![image](https://user-images.githubusercontent.com/6407557/161455040-1c076698-7cba-4bde-9708-4b4d241d4d27.png)

**Comment by davibarreira** [comment]
I think it's actually using the control points to bound... Is this the intended behavior? Causa I was looking for something that bounded the actually object. But I can see how this might be quite complicated.

**Comment by cormullion** [comment]
We could probably iterate over all the curves in the path to find the actual points covered, but I took the easy way out...😀

**Comment by cormullion** [comment]
Hopefully fixed in v3.2

**Comment by davibarreira** [comment]
Awesome! It worked :D
